mne/io/brainvision/brainvision.py

```diff
@@ -535,11 +539,15 @@ def _get_vhdr_info(vhdr_fname, eog, misc, scale, montage):
         for ch in cfg.items('Coordinates'):
             ch_name = ch_dict[ch[0]]
             montage_names.append(ch_name)
-            radius, theta, phi = [float(c) for c in ch[1].split(',')]
             # 1: radius, 2: theta, 3: phi
+            rad, theta, phi = [float(c) for c in ch[1].split(',')]
             pol = np.deg2rad(theta)
             az = np.deg2rad(phi)
-            pos = _sph_to_cart(np.array([[radius * 85., az, pol]]))[0]
+            # Coordinates could be "idealized" (spherical head model)
+            if set(rad) == set([1]):
+                # scale up to realistic head radius (8.5cm == 85mm)
+                rad = [r_ * 85. for r_ in rad]
+            pos = _sph_to_cart(np.array([[rad, az, pol]]))[0]
```

Comment:
These data are the same as we would read in a `BVEF` file, yet when reading via `read_montage`, we do a scaling step:

https://github.com/mne-tools/mne-python/blob/d75bce15717ee4cbf2abbdb5c6e5148959000a91/mne/channels/montage.py#L337-L345

this is not done here, and I can imagine that this is an error.

This has probably not been caught before because very few BrainVision datasets have the "coordinates" section, because there is a tricky step involved when starting the recording (I just learned that myself with the actual hardware a couple of days ago)

# suggestion:
we can scale this to meters as if a unit "auto" were specified

Reply:
fixed in https://github.com/mne-tools/mne-python/pull/6600/commits/35a70bc5de26310bc7a5b1bd2b8774f84b922795

and indeed, this was the solution to the error I stumbled over earlier